ui/pages/first-time-flow/seed-phrase/confirm-seed-phrase/draggable-seed.component.js

```diff
@@ -64,6 +66,7 @@ class DraggableSeed extends Component {
           onClick={onClick}
           data-testid={`draggable-seed-${selected ? 'selected-' : ''}${word}`}
         >
+          {indexShow ? <span className='confirm-seed-phrase__index'>{index + 1}.</span> : null}
```

Comment:
```suggestion
          {indexShow && <span className='confirm-seed-phrase__index'>{index + 1}.</span>}
```

Reply:
fixed in a176d61bfec30de510c149b1d6e22d9b6619a283